docs/Technical Details/index.rst

```diff
@@ -172,8 +177,22 @@ Version Argument
 ``version_argument`` is a way for us to describe how a command requests its version inside of ``easy_infra``. This is useful to know because we avoid running
 security scans (and validation, if any is specified) when the version of a tool is being queried inside of an ``easy_infra`` container.
 
+build/
+======
+
+``Dockerfile``s must all be able to be built independently, as long as their pre-requisites are met. Typically this means you pass in the appropriate
```

Comment:
This is additional documentation, possibly overlapping with `CONTRIBUTING.md`.  Open to de-duplicating